xscen/aggregate.py

```diff
@@ -79,38 +82,233 @@ def climatological_mean(
         Returns a Dataset of the climatological mean
```

Comment:
> I assume you plan on removing everything that is commented ?

Indeed. I left it there for the PR and as a reference, since it was the staring point for climatological_op.

Reply:
`    Returns
    -------
    xr.Dataset
        Returns a Dataset of the climatological mean, by calling climatological_op with option op=='mean'.`

Like that?

Reply:
> I would write in the docs that we are using `climatiological_op`

`    Returns
    -------
    xr.Dataset
        Returns a Dataset of the climatological mean, by calling climatological_op with option op=='mean'.`

Like that?

Reply:
> I assume you plan on removing everything that is commented ?

Yes, removed.

Reply:
actually, I would more put this information (uses clim_op) in the description at the top of the docstring.

Reply:
just saw that it was already done!